tests/unit/handler.unitTest.ts

```diff
@@ -95,7 +95,7 @@ describe("The configuration service", () => {
       process.env.BRANCH = "local";
       const configService = Configuration.getInstance();
       const functions = configService.getFunctions();
-      expect(functions.length).toBe(3);
+
```

Comment:
Why has this check been removed? should we not still have check for the new length of the array?